.changeset/modern-insects-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/theme": patch
+"@wso2is/console": patch
```

Comment:
Since there is a version bump in the theme module, shouldn't we bump myaccount and identity-apps-core packages as well?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5174/commits/cd1a6f3d5c3b5d32b8009333a7981ab7fc29fd38